docs/guides/slashcommands.md

```diff
@@ -5,7 +5,7 @@ authors: Epikcord.py
 
 # Slash Commands
 
-This page will show you how to use Slash Commands with Pycord. 
+This page will show you how to use Slash Commands with EpikCord.py. 
```

Comment:
You don't "use" slash commands with EpikCord.py, that's Discord. You *do* create them though lmao